app/api.py

```diff
@@ -799,20 +799,23 @@ def get_element_text(self, doc, tag_name):
         else:
             return None
 
-    #Send hit to GA
-    def gamp(e,ec):
+    # ga_send_hit sends hit to Google Analytics via Measurment Protocol.
+    # With Measurement Protocol, you can send data by making HTTP requests.
+    # You can find more on developer guide.
+    # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
+    def ga_send_hit(self, event_category):
```

Comment:
"ga" is still acronym, so use "google_analytics" instead. Would be nicer to make it more natural English too e.g., "send_hit_to_google_analytics".